### PR TITLE
Adding select all to the documentslist and a back button to the multiEditThesaurus

### PR DESCRIPTION
**Pull request by rustedgrail**
fixes #

PR checklist:
- [ ] Update READ.me ?
- [ ] Update API documentation ?

QA checklist:
- [ ] Smoke test the functionality described in the issue
- [ ] Test for side effects
- [ ] UI responsiveness
- [ ] Cross browser testing
- [ ] Code review

### CODE REVIEW

File: app/react/Layout/DocumentsList.js [hunk 1]
```diff
@@ -119,6 +130,20 @@ class DocumentsList extends Component {
               stateProperty={this.props.sortButtonsStateProperty}
               storeKey={this.props.storeKey}
             />
+            <div className="select-all-documents">
```
Comment (rustedgrail) [inline app/react/Layout/DocumentsList.js]: The select all works for the other document list views, but should this button only be added on the multiEditThesaurus page?

Comment (bdittes) [reply app/react/Layout/DocumentsList.js]: I think it makes sense for all.

File: app/react/Layout/DocumentsList.js [hunk 2]
```diff
@@ -32,6 +33,13 @@ class DocumentsList extends Component {
     }
   }
 
+  selectAllDocuments() {
+    this.props.documents.get('rows').forEach(doc => {
+      this.ctrlKey = true;
+      this.clickOnDocument(this, doc);
```
Comment (bdittes) [inline app/react/Layout/DocumentsList.js]: Why not simply call this.props.selectDocuments once?

File: app/react/Layout/DocumentsList.js [hunk 3]
```diff
@@ -69,6 +77,9 @@ class DocumentsList extends Component {
       connectionsGroups,
       LoadMoreButton,
       rowListZoomLevel,
+      location: {
```
Comment (bdittes) [inline app/react/Layout/DocumentsList.js]: please use the existing state.library.sidepanel.multiEditOpts instead of reparsing from the query.

Comment (bdittes) [reply app/react/Layout/DocumentsList.js]: Also, I think thesaurusName is not usually populated, so you'll need to look up the id in state.thesauris.find(t => t._id.toString() == opts.thesaurus)

File: app/react/Thesauri/ThesaurusCockpit.tsx [hunk 4]
```diff
@@ -108,6 +108,7 @@ export class ThesaurusCockpitBase extends RouteHandler {
         ? new Date(+modelTime * 1000)
         : null;
     let status;
+    const multiEditUrl = `/library/?multiEditThesaurus=${thesaurus._id}&thesaurusName=${thesaurus.name}`;
```
Comment (bdittes) [inline app/react/Thesauri/ThesaurusCockpit.tsx]: With the changes in DocumentsList, this shouldn't be needed.

**Pull request merged**